Misc/ACKS

```diff
@@ -1900,3 +1900,4 @@ Robert Leenders
 Tim Hopper
 Dan Lidral-Porter
 Ngalim Siregar
+Tim Gates
```

Comment:
For future PRs, please keep the ordering (alphabetical by last name) in this file.  Thanks!

Reply:
Oh sorry  - I didn't realize.